etc/config/c++.defaults.properties

```diff
@@ -50,6 +50,18 @@ compiler.clang12.name=clang 12
 compiler.clangdefault.exe=/usr/bin/clang++
 compiler.clangdefault.name=clang default
 
+group.edg.compilers=edgdefault
+group.edg.compilerType=edg
+group.edg.compilerCategories=edgcpfe
+compiler.edgdefault.exe=/mnt/compiler-explorer/edg-compiler/eccp-scripts/eccp-default
```

Comment:
I'm not sure if this should be changed or not, it's basically just an artifact of how I setup my VM.  Perhaps these paths should be normalized to `/usr/bin/eccp-default`?

Reply:
If that's a "more" usual spot to install it (which it sounds it), then yes please. Or; if you don't think the average CE user will have or want EDG, then feel free not to add it and its configuration here at all (it can live only in a local properties for those that need it, or in the amazon properties only.

That said; having it here at all gives us something to base the Amazon config on, once we agree how to get the compiler to us.